src/components/header/header/header.scss

```
@@ -95,9 +95,9 @@
   padding-inline-start: var(--sbb-spacing-responsive-s);
 }
 
-// Fix left offset if first element of the header is a sbb-header-action.
-// The value of the offset is calculated inside sbb-header-action styles.
-::slotted(sbb-header-action:first-child) {
+// Fix left offset if first element of the header is a sbb-header-button/sbb-header-link.
+// The value of the offset is calculated inside sbb-header-button/sbb-header-link styles.
+::slotted(:is(sbb-header-button:first-child, sbb-header-link:first-child)) {
```

Nitpick (and unsure if it works, due to the nature of ::slotted)
```suggestion
::slotted(:is(sbb-header-button, sbb-header-link):first-child) {
```